packages/discord.js/src/structures/Guild.js

```diff
@@ -1145,6 +1167,21 @@ class Guild extends AnonymousGuild {
     return this.edit({ premiumProgressBarEnabled: enabled, reason });
   }
 
+  /**
+   * Edits the safety alerts channel of the guild.
+   * @param {?TextChannelResolvable} safetyAlertsChannel The new safety alerts channel
+   * @param {string} [reason] Reason for changing the guild's safety alerts channel
+   * @returns {Promise<Guild>}
+   * @example
+   * // Edit the guild safety alerts channel
+   * guild.setSafetyAlertsChannel(channel)
+   *  .then(updated => console.log(`Updated guild safety alerts channel to ${guild.safetyAlertsChannel.name}`))
```

Comment:
For consistency, this should use the `updated` variable:

```suggestion
   *  .then(updated => console.log(`Updated guild safety alerts channel to ${updated.safetyAlertsChannel.name}`))
```